.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,66 @@
+name: test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+env:
+  INGEST_DOCUMENT_BUCKET: ingest-unit-test-document-bucket
+  INGEST_PIPELINE_BUCKET: ingest-unit-test-pipeline-bucket
+  TEST_DATA_FILE_PATH: ./test/data/docs_test_subset.json
+  TEST_DATA_UPLOAD_PATH: input/docs_test_subset.json
+  INGEST_OUTPUT_PREFIX: ingest_unit_test_parser_input
+  DOCUMENT_NAME_KEY: document_id
+  TEST_DATA_EXPECTED_OUTPUT_FILE_PATH: test/data/docs_test_subset_parser_input_expected.json
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setting up Github Actions
+        uses: actions/checkout@v3
+
+      - name: Setting up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+
+      - name: Installing Dependencies
+        run: pip install -r requirements.txt
+
+      - name: Building the Image
+        run: make build_test
+
+      - name: Configuring AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.UNIT_TESTS_AWS_ACCESS_KEY_ID }}
```

Comment:
Quick question - I assume these are stored in Bitwarden? Could we update the readme to not where the secrets are stored :)

Reply:
The entire readme needs an update so we can incorporate that into this PR. I'll do that now.

Reply:
Readme updated.